### DISCUSSION TITLE
Kindle Previewer wrongly set to auto-update

### DISCUSSION THREAD

**Discussion by backstab5983** [Homebrew/discussions#4810] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.1.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: a8519f78fb63f2f2266950bdd8141037da69f8bd
Last commit: 2 days ago
Core tap JSON: 27 Sep 05:28 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: octa-core 64-bit arm_firestorm_icestorm
Clang: 15.0.0 build 1500
Git: 2.39.3 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.1.2 => /usr/bin/curl
macOS: 13.6-arm64
CLT: 15.0.0.0.1.1694021235
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

The cask is set to auto-update but I don't see any feature or option in the app to actually update hence I doubt it has an auto-update feature.

I found this out as the latest version on Brew is 3.72 but 3.73 is out and available and cannot update to it through the app itself.

### Relevant casks

kindle-previewer

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/156289
